tests/datamock/mock_tradeapi.py

```diff
@@ -1,4 +1,5 @@
 from alpaca_trade_api.entity import Asset, BarSet
```

Comment:
Is this import necessary? Looks like only V2 entities are being used?

Reply:
We can remove the import for barset but asset is still being used here